war/src/main/js/keyboard-shortcuts.js

```diff
@@ -0,0 +1,23 @@
+import hotkeys from "hotkeys-js"
+
+window.addEventListener("load", () => {
+  const searchBar = document.querySelector("#search-box")
+  searchBar.placeholder = searchBar.placeholder + ` (${translateKeyboardShortcutForOS("CMD+K")
+    .replace("CMD", "⌘")})`
+
+  hotkeys(translateKeyboardShortcutForOS("CMD+K"), () => {
+    searchBar.focus()
+
+    // Returning false stops the event and prevents default browser events
+    return false
+  })
+})
+
+/**
+ * Translates a given keyboard shortcut, e.g. CMD+K, into an OS neutral version, e.g. CTRL+K
+ * @param {string} keyboardShortcut The shortcut for translation
+ */
+function translateKeyboardShortcutForOS(keyboardShortcut) {
+  const isMac = navigator.platform.toUpperCase().indexOf("MAC") >= 0
```

Comment:
Isn't platform detection that way deprecated?

Reply:
I seem to recall reading that (possibly on MDN) but it doesn't appear to mention that anymore, in fact the only time it recommends using `navigation.platform` is for our exact situation - https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform#examples

Reply:
If MDN is fine with it we can and should use it for sure.